app/packages/app/src/useSetters/onSetFieldVisibilityStage.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  setFieldVisibilityStage,
+  setFieldVisibilityStageMutation,
+  subscribeBefore,
+} from "@fiftyone/relay";
+import { stateSubscription } from "@fiftyone/state";
+import { commitMutation } from "relay-runtime";
+import { pendingEntry } from "../Renderer";
+import { RegisteredSetter } from "./registerSetter";
+
+const onSetFieldVisibilityStage: RegisteredSetter =
+  ({ environment, router, sessionRef }) =>
+  ({ get, set }, input) => {
+    if (!input) return;
```

Comment:
When `input` is not defined, the field visibility stage should be cleared on the server, no?